reportportal_client/service.py

```diff
@@ -203,7 +203,10 @@ def __init__(self,
             self.session.mount('http://', HTTPAdapter(
                 max_retries=retries, pool_maxsize=max_pool_size))
         self.session.headers["Authorization"] = "bearer {0}".format(self.token)
-        self.launch_id = None
+        if kwargs.get('launch_id'):
```

Comment:
```suggestion
        self.launch_id = kwargs.get('launch_id')
```

Reply:
..true..the `.get()` on a `dict` implictly returns `None`, so this should be safe. Thanks!